TailorSort.java

```diff
@@ -43,7 +44,32 @@ private static boolean isArraySorted(Integer[] a){
         //WRITE YOUR CODE HERE
         //RETURN THE RESULT COATS ARRAY WITH ALL SORTED
         //YOU HAVE TO DO MINIMUM COMPARISONS, CLUE: USE index parameter in your sorting technique
+    	Integer key ;
+    	int swapIndex = 0;
+     	for (int i = 0; i < inputCoats.length ;i++ ){
```

Comment:
@UUsha : You are right, Mouly wants the entire array to be sorted. What Mouly means is "Instead trying to sort from the beginning of the array, start from the unsorted part of the array (after the _index_) and try to place them in the right position.